### ISSUE TITLE
[Question] Elaborate on the usage of Kotlin scripting

### ISSUE THREAD

**Issue by esafirm** [DevSrSouza/svg-to-compose#13]
Can you elaborate on how we can execute the scripting mentioned in [the readme](https://github.com/DevSrSouza/svg-to-compose#example-1-running-with-kotlin-scripting)? Thanks

**Comment by c5inco** [comment]
Agreed, the Jitpack declaration doesn't seem to be working, and can't find the respective tagged release in this repo.

**Comment by c5inco** [comment]
You might have to add `#!/usr/bin/env kotlin` to the top of the file. At least that is how I got it to work.
